mlflow/types/llm.py

```diff
@@ -215,6 +215,50 @@ def __post_init__(self):
         self._validate_field("tool_call_id", str, False)
 
 
+@dataclass
+class ChatChunkMessage(_BaseDataclass):
+    """
+    A streaming message chunk in a chat response.
+
+    Args:
+        role (str): The role of the entity that sent the message (e.g. ``"user"``,
+            ``"system"``, ``"assistant"``, ``"tool"``).
+            defaults to ``"assistant"``
+        delta (str): The new token being streamed
+            **Optional** Can be ``None`` if refusal or tool_calls are provided.
+        refusal (str): The refusal message content.
+            **Optional** Supplied if a refusal response is provided.
+        name (str): The name of the entity that sent the message. **Optional**.
+        tool_calls (List[:py:class:`ToolCall`]): A list of tool calls made by the model.
+            **Optional** defaults to ``None``
+        tool_call_id (str): The ID of the tool call that this message is a response to.
+            **Optional** defaults to ``None``
+    """
+
+    role: str = "assistant"
```

Comment:
FYI adding this as I believe it's a reasonable default for streaming chat message chunk responses